packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -1128,6 +1143,59 @@ class GoTrueClient {
     _currentUser = null;
   }
 
+  void mayStartBroadcastChannel() {
+    if (const bool.fromEnvironment('dart.library.html') && _broadcastSession) {
```

Comment:
Not a blocker, but just personally curious. Why are you using this instead of `kIsWeb`?

Reply:
`kIsWeb` is restricted to Flutter. `gotrue_client` runs in Dart apps as well.

Reply:
Ah, right!